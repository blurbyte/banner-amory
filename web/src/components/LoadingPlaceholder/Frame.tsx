// banner and sword wrapper

import styled from 'styled-components';

const Frame = styled.div`
  position: relative;
  width: 15rem;
`;

export default Frame;
