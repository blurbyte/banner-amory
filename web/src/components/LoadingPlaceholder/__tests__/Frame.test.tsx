import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Frame from '../Frame';

test('renders correctly', () => {
  const component = renderer.create(<Frame />);
  expect(component.toJSON()).toMatchSnapshot();
});
