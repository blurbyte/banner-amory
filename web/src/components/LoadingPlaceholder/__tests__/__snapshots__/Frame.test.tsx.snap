// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: relative;
  width: 15rem;
}

<div
  className="c0"
/>
`;
